db/fixtures/ae_datastore/ManageIQ/Cloud/Orchestration/Operations/Methods.class/__methods__/available_vdc_networks.rb

```diff
@@ -0,0 +1,47 @@
+class AvailableVdcNetworks
+  def initialize(handle = $evm)
+    @handle = handle
+  end
+
+  def main
+    fill_dialog_field(fetch_list_data)
+  end
+
+  def fetch_list_data
+    service = @handle.root.attributes["service_template"] || @handle.root.attributes["service"]
+    vdc_networks = service.try(:orchestration_manager).try(:cloud_networks).try(:select) do |net|
+      net.type == "ManageIQ::Providers::Vmware::NetworkManager::CloudNetwork::OrgVdcNet"
+    end
+
+    return nil => "<none>" if vdc_networks.blank?
+
+    vdc_networks_list = { nil => "<select>" }
```

Comment:
I've decided not to choose the default one automatically. It is still possible not to choose the network in which case the vCloud will not connect the VMs to any network. This is also the way it works on the vCloud Director where no network is selected by default.